### ISSUE TITLE
Stop the agent instead of changing lane

### ISSUE THREAD

**Issue by YashNamdeo** [thibo73800/metacar#37]
How to stop the agent whenever it's path is blocked or let it just simply follow other vehicles instead of changing lanes? I tried various things but nothing is reflecting while training.

**Comment by thibo73800** [comment]
Hi, you want the agent to learn a behavior to follow other vehicles rather than changing lanes ?

**Comment by YashNamdeo** [comment]
Yes, I want the agent to stop whenever any other vehicle comes on its way or simply follow the other vehicles without undergoing any collision. 
So basically I just want to introduce a braking part in it.
The agent should accelerate whenever the path is free and at the same time it should reduce the speed or it should stop whenever it's path is blocked but it should never change lane.

I have tried various things ranging from changing the control motion to changing reward function but still nothing seems to work out.
